rosdep/python.yaml

```diff
@@ -2656,6 +2656,22 @@ python-pyramid:
   fedora: [python-pyramid]
   gentoo: [dev-python/pyramid]
   ubuntu: [python-pyramid]
+python-pyshark-pip:
+  debian:
+    pip:
+      packages: [pyshark]
+  fedora:
+    pip:
+      packages: [pyshark]
+  gentoo:
+    pip:
+      packages: [pyshark]
+  ubuntu:
+    pip:
+      packages: [pyshark]
+  osx:
+    pip:
+      packages: [pyquery]
```

Comment:
This is surprising to have a different pip name on osx. Can you link to a resource that explains why this is different?